### DISCUSSION TITLE
Welcome to Species Occurrence (SPOC) Discussions!

### DISCUSSION THREAD

**Discussion by jermnelson** [sul-dlss-labs/spoc#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amandawhitmire** [comment]
Hi, I'm Amanda and I'm the librarian "subject expert" with this project. I have a background in biological oceanography. I am new to this level of computational text analysis and app development, and am very excited to be learning through the SPOC project.
